MANIFEST.in

```diff
@@ -1,2 +1,23 @@
-recursive-include shoop *.jinja *.html *.txt
-recursive-include shoop_tests *.jinja *.html *.txt
+include README.rst
+include LICENSE
+include VENDOR-LICENSES.md
+include ChangeLog.md
+
+include tox.ini
+
+include build_resources.py nodejs_verify.py
```

Comment:
Nope. Just to sdist. Not really sure why, though. Maybe because they're not inside any python package?